Plugins/NativeCapture/Source/NativeCapture.cpp

```diff
@@ -0,0 +1,81 @@
+#include "NativeCapture.h"
+
+#include <Babylon/JsRuntime.h>
+#include <GraphicsImpl.h>
+
+#include <vector>
+
+namespace Babylon::Plugins::Internal
+{
+    class NativeCapture : public Napi::ObjectWrap<NativeCapture>
+    {
+    public:
+        static constexpr auto JS_CLASS_NAME = "NativeCapture";
+
+        static void Initialize(Napi::Env env)
+        {
+            Napi::HandleScope scope{env};
+
+            Napi::Function func = NativeCapture::DefineClass(
+                env,
+                JS_CLASS_NAME,
+                {
+                    NativeCapture::InstanceMethod("addOnCaptureCallback", &NativeCapture::AddOnCaptureCallback),
```

Comment:
nit: AddOnCaptureCallback or AddCaptureCallback?

Reply:
Is your nit about casing? I'm confused because the current casing looks right for JS, so maybe I'm missing something.

Reply:
No, it's whether the "On" part should be there or not.